web/packages/design/src/utils/copyToClipboard.ts

```diff
@@ -15,13 +15,19 @@
  */
 
 /**
- * copyToClipboard copies text to clipboard.
+ * Copies text to clipboard.
  *
  * @param textToCopy the text to copy to clipboard
  */
-export default function copyToClipboard(textToCopy: string): Promise<any> {
-  return navigator.clipboard.writeText(textToCopy).catch(err => {
-    // This can happen if the user denies clipboard permissions:
-    window.prompt('Cannot copy to clipboard. Use ctrl/cmd + c', err);
```

Comment:
I think we wanted to use `window.alert` here (which works in Electron btw).

Reply:
I think it was actually meant to be `window.prompt` with `textToCopy` as the second argument. This way if `navigator.clipboard.writeText` failed, the browser would show a prompt with the input prefilled. Then you can simply copy the contents, which might be already preselected.

Reply:
Makes sense, I didn't think of it! 
So for Web UI we will use `window.prompt` and for Connect `window.alert`.